examples/embeddable_examples/public/app/state_management_example/state_management_example.tsx

```diff
@@ -123,13 +123,13 @@ export const StateManagementExample = ({ uiActions }: { uiActions: UiActionsStar
         <EuiFlexItem grow={false}>
           <EuiButton
             disabled={isSaving || !hasUnsavedChanges}
-            onClick={async () => {
+            onClick={() => {
               if (!bookApi) {
                 return;
               }
 
               setIsSaving(true);
-              const bookSerializedState = await bookApi.serializeState();
+              const bookSerializedState = bookApi.serializeState();
               if (!isMounted()) {
```

Comment:
isMounted check can be removed since callback is now sync

Reply:
Addressed and some other cleanups in 8b0809cccd7